packages/ann/ann/c_src/logistic_actf_component.h

```diff
@@ -46,6 +46,26 @@ namespace ANN {
 
     virtual char *toLuaString();
   };
+
+  class SparseLogisticActfANNComponent: public LogisticActfANNComponent {
+
+    private:
+       float beta;
```

Comment:
The following had to be indented two spaces more than class keyword.
https://github.com/pakozm/april-ann/blob/master/STYLEGUIDE.md